test/unittests/testData/ngsi10.restReply.updateContextResponse01.ok.valid.xml

```diff
@@ -2,5 +2,6 @@
   <errorCode>
     <code>404</code>
     <reasonPhrase>No context element found</reasonPhrase>
+    <details>detail</details>
```

Comment:
This is adding info. Can't imagine anything will break for adding info. And if it does, it's their own problem, I'd say.
However, this is from a unit_test, creating the structures by hand. Not sure an external user will ever experience any difference

NTC